libqtile/core/manager.py

```diff
@@ -195,6 +157,59 @@ def noop(qtile):
 
         hook.fire("startup_complete")
 
+    def load_config(self, config):
+        try:
+            config.load()
+        except Exception as e:
+            logger.exception('Error while reading config file (%s)', e)
+            config = confreader.Config()
+            from libqtile.widget import TextBox
+            widgets = config.screens[0].bottom.widgets
+            widgets.insert(0, TextBox('Config Err!'))
+
+        self.core.wmname = getattr(config, "wmname", "qtile")
+        if config.main:
+            warnings.warn("Defining a main function is deprecated, use libqtile.qtile", DeprecationWarning)
+            config.main(self)
+
+        if config.groups:
+            key_binder = None
+            if hasattr(config, 'dgroups_key_binder'):
+                key_binder = config.dgroups_key_binder
+            self.dgroups = DGroups(self, config.groups, key_binder)
+
+        if hasattr(config, "widget_defaults") and config.widget_defaults:
+            _Widget.global_defaults = config.widget_defaults
+        else:
+            _Widget.global_defaults = {}
+
+        if hasattr(config, "extension_defaults") and config.extension_defaults:
+            _Extension.global_defaults = config.extension_defaults
+        else:
+            _Extension.global_defaults = {}
+
+        for installed_extension in _Extension.installed_extensions:
+            installed_extension._configure(self)
+
+        for i in self.groups:
+            self.groups_map[i.name] = i
+
+        for grp in config.groups:
+            if isinstance(grp, ScratchPadConfig):
+                sp = ScratchPad(grp.name, grp.dropdowns, grp.label)
+                sp._configure([config.floating_layout],
+                              config.floating_layout, self)
+                self.groups.append(sp)
+                self.groups_map[sp.name] = sp
+
+        # It fixes problems with focus when clicking windows of some specific clients like xterm
+        # 2020-12-31: unsure if this still fixes anything
```

Comment:
Can we just drop it and see if people complain? Either that or drop this additional comment. I don't think it adds any value.

Reply:
wrong PR =)